server/tests/backends/fixtures/ifthenelse/dates_conditions_pypika.json

```
@@ -0,0 +1,121 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "ifthenelse",
+        "newColumn": "cond",
+        "if": {
+            "column": "brewing_date",
+            "operator": "from",
+            "value": "2022-01-05 00:00:00"
+        },
+        "then": "'tintin'",
+        "else": {
+          "if": {
+              "column": "brewing_date",
+              "operator": "until",
+              "value": "2022-01-07 00:00:00"
```

```suggestion
              "value": "2022-01-03 00:00:00"
```

We'll never get `tata` with a date later than the date in the first predicate